specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -147,6 +157,49 @@ export function TreeRow({
   const hasNoChildrenNodes =
     nodeStats?.directCount === 0 && nodeStats.childCount === 0;
 
+  const [resource] = useLiveState<SpecifyResource<AnySchema> | undefined>(
+    React.useCallback(() => {
+      const table = genericTables[treeName] as SpecifyTable<AnyTree>;
+      return new table.Resource({ id: row.nodeId });
+    }, [row.nodeId, treeName])
+  );
+
+  const [loadedResource] = useAsyncState(
+    React.useCallback(async () => {
+      if (resource === undefined) return;
+      return hijackBackboneAjax(
+        [Http.NOT_FOUND],
+        async () => resource.fetch(),
+        (status) =>
+          status === Http.NOT_FOUND ? unsafeTriggerNotFound() : undefined
+      );
+    }, [resource]),
+    false
+  );
+
+  const acceptedChildren = loadedResource?.get('acceptedChildren') || '';
+
+  const fetchedChildren = async (): Promise<any> => {
```

Comment:
@realVinayak could you please advise if it would be better to do this on the back-end (when the rows themself are fetched) or on the front-end here (which would create a lot of extra network requests)?